### ISSUE TITLE
Missing accessible labels for contact detail properties

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/contacts#5048]
### Steps to reproduce

1. Go to contact details
2. Find any fields

### Expected behavior

They should have a label connected to the input field

### Actual behavior

The label is a heading, disconnected from the input field.

Adding placeholders is not enough to make this accessible

### PHP engine version

None

### Web server

None

### Database

None
